### PR TITLE
README update

### PR DESCRIPTION
**Pull request by natke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,133 +2,140 @@
...
+* Supported sampling features
+  * Beam search
+  * Greedy search
+* APIs
+  * Python
+  * C/C++  
+
+## Coming very soon
+
+* Support for the Mistrel and Whisper model architectures
```
Comment (RyanUnderhill) [inline README.md:66]: Mistrel -> Mistral?
For Whisper, you just mean supporting longer than 30 second audio streams?

File: README.md [hunk 2]
```diff
@@ -2,133 +2,140 @@
...
+## Features
+
+* Supported model architectures:
+  * Phi-2
+  * Llama
+  * GPT
+* Supported targets:   
+  * CPU
+  * GPU (CUDA)
+* Supported sampling features
```
Comment (RyanUnderhill) [inline README.md:57]: Should we say Top/TopK/TopP?

File: README.md [hunk 3]
```diff
@@ -2,133 +2,140 @@
...
+
+## Model download and export
+
+ONNX models are run from a local folder, via a string supplied to the `Model()` method. 
+
+To source `microsoft/phi-2` optimized for your target, download and run the following script:
+
+```bash
+wget https://github.com/microsoft/onnxruntime-genai/blob/kvaishnavi/models/models/export.py
+```
```
Comment (RyanUnderhill) [inline README.md:132]: I think Kunal is about to check this in

File: README.md [hunk 4]
```diff
@@ -2,133 +2,140 @@
...
+   git clone https://github.com/microsoft/onnxruntime-genai
+   ```
+
+2. Install ONNX Runtime
+
+   ```bash
+   mkdir -p ort/include
+   cd ort/include
+   wget https://raw.githubusercontent.com/microsoft/onnxruntime/v1.17.0/include/onnxruntime/core/session/onnxruntime_c_api.h
+   wget https://raw.githubusercontent.com/microsoft/onnxruntime/v1.17.0/include/onnxruntime/core/session/onnxruntime_cxx_api.h
```
Comment (RyanUnderhill) [inline README.md:91]: the onnxruntime_cxx_api.h and onnxruntime_cxx_inline.h files aren't needed, only onnxruntime_c_api.h

File: README.md (hunk 1 above)
Comment (natke) [reply README.md:66]: No I mean support for Whisper (e2e)

File: README.md (hunk 2 above)
Comment (natke) [reply README.md:57]: Added

File: README.md (hunk 3 above)
Comment (natke) [reply README.md:132]: He says he needs to do some more work

**Review by RyanUnderhill — APPROVED** [review]

**Pull request merged**